### PR TITLE
Add custom agreement links for login page

### PR DESCRIPTION
**Pull request by eitan-kr**

### CODE REVIEW

File: charts/akeyless-api-gateway/values.yaml [hunk 1]
```diff
@@ -433,6 +433,10 @@ metrics:
 # - logand-conf field (base64). The content the same as above.
 logandExistingSecretName:
 
+# Specifies custom agreement links to appear on login page
+loginPageAgreementLinks:
+#  endUserLicenseAgreement:
```
Comment (niv-ez) [inline charts/akeyless-api-gateway/values.yaml:438]: please provide an example for input here with explanation , and a link to akeyless documentation

Comment (eitan-kr) [reply charts/akeyless-api-gateway/values.yaml:438]: Added example.
Explanation is in line 436.
No documentation yet, as this is a new feature not yet deployed.

**Comment by eitan-kr** [comment]
@niv-ez  Tested building with helm

**Comment by niv-ez** [comment]
@eitan-kr please update your branch and chart version

**Review by niv-ez — APPROVED** [review]

**Review by rani-sharim — APPROVED** [review]

**Review by omriezra — APPROVED** [review]

**Pull request merged**